happypose_ros/test/single_view_base.py

```diff
@@ -98,17 +121,17 @@ def test_03_trigger_pipeline(self, proc_output: ActiveIoHandler) -> None:
             ready = proc_output.waitFor("HappyPose initialized", timeout=0.5)
         assert ready, "Failed to trigger the pipeline!"
 
-    def test_04_receive_messages(self) -> None:
+    def test_06_receive_messages(self) -> None:
```

Comment:
Why numbering?

Reply:
It enforces order of the unit tests as they are executed alphabetically. This is due to the sequential nature of the way the ROS node works and in this case, actions have to be executed in a certain order to avoid repeating of computations, which are costly in this case